AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -281,6 +281,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/100355
+noltrt.com#%#//scriptlet('adjust-setTimeout', '/\(\'#download_ad_addon\'\)\.slideUp/', '10000', '0.02')
```

Comment:
Is the Regex necessary? Please check `noltrt.com#%#//scriptlet('adjust-setTimeout', '#download_ad_addon', '10000', '0.02')`.

Reply:
Okay.